gravitee-apim-portal-webui-next/src/guards/redirect.guard.ts

```diff
@@ -0,0 +1,14 @@
+import { inject } from '@angular/core';
+import { ConfigService } from '../services/config.service';
+import {Router} from "@angular/router";
+
+export const redirectGuard = (): boolean => {
+  let enabled = inject(ConfigService).portalNext.access?.enabled;
+
+  console.log("Hello")
+  if (!enabled) {
+    console.log("Hello")
+    inject(Router).navigateByUrl("/404/");
```

Comment:
We discussed it on our team we found the solution that we would redirect to "404" which current router will as " component: NotFoundComponent".

Reply:
Yes I was saying that you can use the `navigate` method instead of `navigateByUrl` if you dont use an URL 😅

Reply:
Well, we need to navigate to a non existing page.
My idea is that "/404/" URL is a non existent URL.

If I don't understand your suggestion coirrentcly, can you please suggest that to put inside 
` inject(Router).navigate(?)` so that it would be a non-existent page?

After discussion with out team, any page not a child to "/catalog/api" will be caught with 
`path: '**',
    component: NotFoundComponent,`

Reply:
Resolved with @phiz71